rotkehlchen/chain/bitcoin/hdkey.py

```diff
@@ -398,6 +406,11 @@ def derive_child(self, idx: Union[int, str]) -> 'HDKey':
         return self._child_from_xpub(index=index, child_xpub=child_xpub)
 
     def address(self) -> BTCAddress:
+        if self.hint == 'xpub' and self.xpub_type == XpubType.P2TR:
+            return pubkey_to_bech32_address(
+                data=self.pubkey.format(COMPRESSED_PUBKEY),
+                witver=1,
```

Comment:
what is this and why is it always constant? If it's a constant `1` then define it somewhere along with an explanation of what it is and use it here